README.md

```diff
@@ -66,7 +67,7 @@ const App = React.createClass({
 });
 
 
-React.render(<App />, document.body);
+ReactDOM.render(<App />, appRoot);
```

Comment:
That makes sense. I guess since we are discouraged to render into body itself (which we've never done in real projects anyway), there is a fair bit of boilerplate code. Not good for examples at all. So better to remove this part.